src/components/AccountPage.tsx

```diff
@@ -2,18 +2,18 @@ import * as React from "react";
 import { Store } from "redux";
 import * as PropTypes from "prop-types";
 import ChangePasswordForm from "./ChangePasswordForm";
-import { State } from "../reducers/index";
+import { RootState } from "../store";
 import Header from "./Header";
 import Footer from "./Footer";
 import title from "../utils/title";
 
 export interface AccountPageContext {
-  editorStore: Store<State>;
+  editorStore: Store<RootState>;
```

Comment:
Not a blocker, but `ContextProvider` is naming this `editorStore`, which may lead one to believe that it is just the "editor" half of the store... which I guess is what someone thought it was when they typed it the way they did. So maybe `rootStore` would be more appropriate. But that would require a bunch of changes, and this is still an improvement.

Reply:
Thanks, @ray-lee. I saw that, which I should've mentioned. The risk mitigator here, based on my sampling, is that the components that are using that name really are using only the `editor` portion of the store. I am envisioning that most, if not all of these should eventually be eliminated by introducing slices, which I believe is a much nicer and more maintainable approach.